### PR TITLE
RFC: global authentication (RFC-23)

### PR DESCRIPTION
**Pull request by tbwiss**
# Description

RFC for global authentication.

### CODE REVIEW

File: docs/rfcs/rfc-021-global-authentication.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        globalAuthentication: true,
+    },
+    typeDefs: typeDefs,
+    driver,
+    plugins: {
+        auth: new Neo4jGraphQLAuthJWTPlugin({
+            secret: "1234",
+            noVerify: false,
+        }),
+    },
```
Comment (darrellwarde) [inline docs/rfcs/rfc-021-global-authentication.md:25]: Just a quick parting comment on this before I head off on vacation - I believe that the configuration for global authentication should be in the auth plugin, and not in the library. It depends on JWT decoding and verification, which is a plugin feature.

**Review by darrellwarde — CHANGES_REQUESTED** [review]
A quick comment here!

File: docs/rfcs/rfc-021-global-authentication.md (hunk 1 above)
Comment (tbwiss) [reply docs/rfcs/rfc-021-global-authentication.md:25]: Really good point @darrellwarde! Will make the necessary adjustments

**Comment by tbwiss** [comment]
@darrellwarde after today's talk (we go for the proposed solution), is this ready to be merged?

**Review by darrellwarde — DISMISSED** [review]
Indeed, I think this is good to merge for now!

**Pull request merged**